doc/tut1.rst

```diff
@@ -1441,31 +1441,111 @@ string that is "useless" and replace it with "useful".
 Note: alternate ways of writing this are ``b[^8..^2] = "useful"`` or
 as ``b[11..b.len-2] = "useful"`` or as ``b[11..<b.len-1] = "useful"``.
 
+Objects
+-------
+
+The default type to pack different values together in a single
+structure with a name is the object type. An object is a value type,
+which means that when an object is assigned to a new variable all its
+components are copied as well.
+
+Each object type ``T`` has a constructor ``T(field: value, ...)``
```

Comment:
Since `T` is usually used for generic types (`[T]`), I would change it to something else here, just to avoid any potential confusion.

`O`, `Obj`, `Foo`, or something along these lines should be fine.

Reply:
good point